ext/client.go

```diff
@@ -17,7 +17,7 @@ type Client interface {
 	GetNode(string) (*pfmodel.Node, error)
 	GetContainers() (*pfmodel.ContainerList, error)
 	GetContainer(string) (*pfmodel.Container, error)
-	CreateContainer(string, string, string, string) (*pfmodel.Container, error)
+	CreateContainer(string, string, string, string, string, string, string) (*pfmodel.Container, error)
```

Comment:
this is code smell, need to be extracted into data structure, for example: source
ideally CreateContainer should only have (hostname string, source) as attributes

Reply:
noted.